bindings/matrix-sdk-ffi/src/client.rs

```diff
@@ -243,6 +244,12 @@ impl Client {
 #[uniffi::export(async_runtime = "tokio")]
 impl Client {
     /// Login using a username and password.
+
```

Comment:
```suggestion
///
```

otherwise this would ignore the first part of the doc comment, I suppose (and at least it's more consistent with our styling)